### DISCUSSION TITLE
Request for domain addition

### DISCUSSION THREAD

**Discussion by rounakchourasia17** [RosettaCommons/rosetta#251] [ROSIE] [answered]
I am a Research Associate at National Agri-food Biotechnology Institute (NABI) India. I need to use the FLexPepDock server for academic purposes.
I request you to kindly add the institute as an academic organization. My mail Id is: rounak.ch@nabi.res.in
The web address of the institute is: https://nabi.res.in/

**Comment by lyskov** [comment, marked as answer]
— Thank you for letting us know! This should be fixed now, if you still have issues logging-in please feel free to re-open this issue.
